src/app/pages/jails/jail-wizard/jail-wizard.component.ts

```diff
@@ -54,17 +56,19 @@ export class JailWizardComponent {
           type: 'checkbox',
           name: 'dhcp',
           placeholder: T('DHCP autoconfigure IPv4'),
-          tooltip: T('Check this to automatically configure IPv4 settings\
- for the jail. <b>VirtIO</b> must also be enabled.'),
-        },
+          tooltip: T('Automatically configure the jail to use IPv4 \
+                      networking.\
+                      <br><b>VirtIO</b> must also be enabled.'),
+      },
         {
           type: 'checkbox',
           name: 'vnet',
           placeholder: T('VirtIO Virtual Networking'),
-          tooltip: T('Check to use VirtIO to emulate network devices for\
- this jail. See <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=virtio&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">VIRTIO(4)</a> for more details.'),
+          tooltip: T('Use VirtIO to emulate network devices for the \
+                      jail. <br> \
+                      See <a href="https://www.freebsd.org/cgi/man.cgi?\
```

Comment:
Don't know if it is a standard or not, but we've usually been wrapping links around the `<a` and `target="_blank` spots:
`See <a\
 href="https://www.freebsd.org/cgi/man.cgi?query=virtio&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
 target="_blank">VIRTIO(4)</a> for more details.`

This seems to balance the readability and line length pretty well. It also makes it easier to select and replace links.

Reply:
Sorry, github is trying to help with the formatting - message me on Rocket and I can get you an example.

Reply:
This was 1 of my first tooltip reviews and didn't realize how we were wrapping URLs yet. The rest of my URLs are wrapped properly.